package.json

```diff
@@ -31,7 +31,8 @@
     "dependencies": {
         "camelcase": "^4.1.0",
         "camelcase-keys": "^4.1.0",
-        "filter-obj": "^1.1.0"
+        "filter-obj": "^1.1.0",
+        "is-wsl": "1.1.0"
```

Comment:
Indeed sorry, my default NPM settings are --save-exact=true. I'll change it.

Reply:
Yeah, I figured. I had that for a while, too. Gave it up because it is a false sense of security. It only locks down your top level dependencies. If any of those dependencies have semver ranges and a sub-dependency violates semver (or has other issues), you're probably still going to be exposed to it. Updates tend to be good, anyway. And when you really need to lock things down properly, there is shrinkwrap.

Reply:
Yeah I might reconsider it indeed.